test/rules/structured-imports/ancestorCases.ts.lint

```diff
@@ -0,0 +1,11 @@
+import * as relative from "./relativeImport1";
```

Comment:
Tests should also verify what happens with side-effect imports (e.g., `import './App.scss'`).

No idea what should be done with those, but we should make sure nothing's breaking when they're present 😄